src/openrct2/interface/window.h

```diff
@@ -36,6 +36,7 @@ struct TitleSequence;
 struct TextInputSession;
 
 #define TEXT_INPUT_SIZE 1024
+#define TOP_TOOLBAR_CLEARANCE 28
```

Comment:
I've never heard of 'Clearance' in UI design. 'Keyline' maybe, but height seems like the most fitting term.